.github/workflows/build.yml

```diff
@@ -23,5 +24,19 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install tox tox-gh-actions
+      - name: Install System Dependencies (macOS)
+        if: runner.os == 'macOS'
+        run: |
+          brew install pkg-config
+          brew install libffi
+          brew install pango
+          brew install glib
+      - name: Install System Dependencies (Windows)
+        if: runner.os == 'windows'
+        run: |
+          curl -L https://github.com/naveen521kk/pango-build/releases/download/v1.48.0/pango-build-win64.zip -o pango.zip
```

Comment:
This will work from next cairocffi release. Because I made a PR recently for adding this new name to it.